packages/experiments/src/components/Stack/StackItem/StackItem.tsx

```diff
@@ -5,17 +5,18 @@ import { IStackItemComponent, IStackItemProps, IStackItemSlots } from './StackIt
 import { styles } from './StackItem.styles';
 
 const view: IStackItemComponent['view'] = props => {
-  const childNodes: React.ReactElement<{}>[] = React.Children.toArray(props.children) as React.ReactElement<{}>[];
+  const { children } = props;
+  const childNodes: React.ReactElement<{}>[] = React.Children.toArray(children) as React.ReactElement<{}>[];
   const first = childNodes[0];
   if (!first) {
     return null;
   }
```

Comment:
I know it's not directly part of this PR, but I'm not sure I understand what lines 9-13 are doing, particularly since I don't see `childNodes` being used anywhere else. Could this instead simply be: 

```tsx
if (React.Children.count(children) < 1) {
    return null;
}
```

Reply:
Yeah, we should probably just do that.